phoenix-core/src/test/java/org/apache/phoenix/index/IndexUpgradeToolTest.java

```diff
@@ -27,34 +27,57 @@
 import org.apache.commons.cli.CommandLine;
 import org.apache.phoenix.mapreduce.index.IndexUpgradeTool;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
+import org.mockito.Mock;
+import org.mockito.Mockito;
 
 @RunWith(Parameterized.class)
 public class IndexUpgradeToolTest {
     private static final String INPUT_LIST = "TEST.MOCK1,TEST1.MOCK2,TEST.MOCK3";
     private final boolean upgrade;
+    private static final String VERIFY_VALUE = "AFTER";
```

Comment:
Nit/TDD guidance:  I would keep the value as "SomeVerifyMode" instead of "AFTER" .. then the test ensures the tool is not actually checking against one of the valid verification modes. Also, highlights the flip side that If the tool is supposed to validate the modes then we have missing tests.